scalyr_agent/platform_posix.py

```diff
@@ -345,9 +347,9 @@ def logger(message):
             # redirect standard file descriptors
             sys.stdout.flush()
             sys.stderr.flush()
-            si = file(self.__stdin, "r")
-            so = file(self.__stdout, "a+")
-            se = file(self.__stderr, "a+", 0)
+            si = open(self.__stdin, "r")
```

Comment:
I am afraid `file` built-in was removed from python3. Moreover, as we discussed earlier, we do not have any other option, rather than use io.open in python3, so it will be better to keep behaviour on python2 closer to the python3. Honestly, I can not find any reason why open could be worse than io.open in terms of getting file info.

Reply:
Ok, but we do have to close `si`, `so`, and `se` now correct?

Reply:
Please correct me if I am wrong.

I think it does not matter how files were opened. We should close them anyway.